### PR TITLE
[readtimer] Separate strings for singular and plural

### PR DESCRIPTION
**Pull request by robert00s**
Close: #3397

### CODE REVIEW

**Comment by Frenzie** [comment]
Thanks!

File: plugins/readtimer.koplugin/main.lua [hunk 1]
```diff
@@ -41,6 +41,10 @@ function ReadTimer:unschedule()
 end
 
 function ReadTimer:addToMainMenu(menu_items)
+    local hour_string = _("hour")
+    local hours_string = _("hours")
```
Comment (Frenzie) [inline plugins/readtimer.koplugin/main.lua]: I thought about this yesterday but I decided to merge it anyway, but today in Transifex I realized I should've listened to my instincts.

Basically, longer, more complete sentences (and yes, that means duplication) are better for localization.

In Dutch the regular plural of a word is on *-en*, and indeed the plural form or *uur* (hour), *uren* does exist, but usually it should be the same.

So you can't just translate "hours" by itself and expect it to match up with whatever you might want to combine it with.

tl;dr I think you shouldn't be programmatically clever for localization. :-)
